File: src/main/java/za/co/ioagentsmith/game/kalaha/controller/LandingPageController.java
package za.co.ioagentsmith.game.kalaha.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class LandingPageController {

    private static final String INDEX = "index";

    @GetMapping("/")
    public String index() {
        return INDEX;
    }
}

File: kalaha-kotlin-backend/src/main/java/za/co/ioagentsmith/game/kalaha/controller/LandingPageController.kt
package za.co.ioagentsmith.game.kalaha.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class LandingPageController {

    @GetMapping("/")
    fun index(): String {
        return INDEX
    }

    companion object {

        internal val INDEX = "index"
    }
}